skrifa/src/util.rs

```diff
@@ -0,0 +1,151 @@
+//! Internal helpers.
```

Comment:
I think this is a useful addition. I'd suggest to rename this to smallarray.rs to avoid generic names such a `util` or `helper` and land this in a separate CL.

Reply:
What about `util::smallarray`? It’s really common in rust to shove these small internal implementation details into a generic module like this to avoid polluting the crate root. For me personally, it makes browsing the code easier because I can distinguish between “real” code and ancillary types at a glance.

Happy to move this to a separate PR.

Reply:
Moved to `small_array` mod and merged in #445